ang/civicase/ActivitiesCalendar.js

```diff
@@ -407,15 +463,27 @@
     }
 
     /**
-     * Loads the dates with at least an activity with the given status(es)
+     * Loads the days within the month of the given date
+     * with at least an activity with the given status(es)
+     *
+     * The days are returned in an object containing also the year+month they
+     * belong to, so that they can be properly grouped in the internal list of days
      *
-     * @param {*} statusParam
+     * @param {*} status
+     * @param {Date} date
      * @return {Promise}
      */
-    function loadDaysWithActivities (statusParam) {
+    function loadDaysWithActivities (status, date) {
```

Comment:
shouldn't the date be the first parameter? `loadDays......('completed', '2000-01-01');`  does not make much sense to me.

Reply:
Also, the first parameter for `getDaysWithActivities` is a date, so it's better to be consistent